fuzz/meson.build

```diff
@@ -0,0 +1,40 @@
+standaloneengine = static_library('standaloneengine',
+    'StandaloneFuzzTargetMain.c',
+    dependencies: libvips_deps,
+)
+
+fuzz_progs = [
+    'jpegsave_file_fuzzer',
+    'jpegsave_buffer_fuzzer',
+    'pngsave_buffer_fuzzer',
+    'webpsave_buffer_fuzzer',
+    'gifsave_buffer_fuzzer',
+    'sharpen_fuzzer',
+    'thumbnail_fuzzer',
+    'smartcrop_fuzzer',
+    'mosaic_fuzzer'
+]
+
+fuzz_execs = []
+foreach fuzz_prog : fuzz_progs
+    fuzz_execs += executable(fuzz_prog,
+        fuzz_prog + '.cc',
+        dependencies: libvips_dep,
+        link_with: standaloneengine
+    )
+endforeach
+
+test_fuzz = configure_file(
+    input: 'test_fuzz.sh',
+    output: 'test_fuzz.sh',
+    copy: true,
```

Comment:
Do you actually need this file? It does nothing other than define $ASAN_OPTIONS $UBSAN_OPTIONS $VIPS_WARNING and then run each of fuzz_execs in turn. You could easily implement that directly in meson. There's an `env` kwarg:

https://mesonbuild.com/Reference-manual_functions.html#test

Reply:
it might not be needed but I'm starting with a 1:1 port and we can simplify it once it is done :smiley: